src/Run/Run.tsx

```diff
@@ -119,7 +119,7 @@ const Run = (props) => {
                 res += ",";
             }
             if (tuple[i]["type"] === "Tuple") {
-                res += "(" + get_tuple(tuple[i]["value"]) + ")";
+                res += get_tuple(tuple[i]["value"]);
```

Comment:
The surrounding parens are already added outside of the conditional.

Reply:
Ahh, well that was easy then 👍